docs/requirements.txt

```diff
@@ -4,7 +4,7 @@ sphinxcontrib-spelling
 Flask-Sphinx-Themes
 
 # code dependencies, needed for imports
-sqlalchemy>=0.9
+sqlalchemy>=0.9,<1.4.0
```

Comment:
This doesn't look like it should be part of this PR

Reply:
I fixed the problems with the automated tests. @kerryhatcher, if you rebase on top of the latest version of the `main` branch, it should work!